src/pip/_internal/basecommand.py

```diff
@@ -151,15 +151,21 @@ def main(self, args):
                 "console": {
                     "level": level,
                     "class":
-                        "pip._internal.utils.logging.ColorizedStreamHandler",
+                    "logging.StreamHandler"
```

Comment:
I'd prefer to have these be conditionals before assignment to the dictionary.

```py
if options.no_color:
    logger_class = "logging.StreamHandler"
else:
    logger_class = "pip._internal.utils.logging.ColorizedStreamHandler"
```

Reply:
This is indeed much better, and I fixed this.